### DISCUSSION TITLE
How can I update a dependency( im newbie =(( )

### DISCUSSION THREAD

**Discussion by lichdu29** [renovatebot/renovate#8174] [Request Help] [answered]
for example i was suggested to do this:
Update dependency handlebars to v4.7.6
Schedule: ["at any time"]
Branch name: renovate/handlebars-4.x
Merge into: master
Upgrade handlebars to 4.7.6

how can i config this in renovate.json file, what should i supposed to edit Please help

**Comment by rarkins** [comment]
If you install Renovate then it should automatically detect and update all your dependencies. Please provide an example repo if not

**Comment by HonkingGoose** [comment, marked as answer]
Found the config @lichdu29 is talking about: https://github.com/lichdu29/LinuxProject/commits/master/renovate.json.

The Renovate bot did raise a update for Handlebars (https://github.com/lichdu29/LinuxProject/pull/3/files), but it's probably missing updates to other dependencies (malformed `package.json`).

There are some problems with the repository's `package.json` which I think is preventing the Renovate bot from making some updates.
I raised a PR to fix their `package.json` so that the Renovate bot can work properly. 😄 

And as an added bonus, went ahead and raised a PR to get them a nice Renovate Dashboard as well. 😄

**Comment by HonkingGoose** [comment]
Found the main problem, @lichdu29 had merged his `master` branch into the `renovate/handlebars-4.x` PR branch, that led to a malformed `package.json` (https://github.com/lichdu29/LinuxProject/pull/3). This malformed `package.json` was then merged into `master` when the Renovate PR got merged.

The lesson here is that you don't need to edit the `renovate/` pull request branches yourself, just let the bot do it's thing. 😉
